cmd/podman/run.go

```diff
@@ -213,8 +212,6 @@ func runCmd(c *cli.Context) error {
 			} else {
 				exitCode = ctrExitCode
 			}
-		} else {
-			logrus.Errorf("Unable to get exit code of container %s: %q", ctr.ID(), err)
```

Comment:
I'm not too clear on where `exitCode` gets initialized, but could removing this leave `exitCode` in an undefined state?

Reply:
exitCode is initialized to 125 in the main.go.  This it the state that is supposed to be returned when podman has a problem.
126 indicates that the container command did not exist. 127 indicates that their was a permission denied when the container command was attempted to be started.  All other exist codes should be from the command executing inside of the container.

Reply:
125 also implies the container failed, which it hasn't - it's still running.
This also won't fix restarted containers, which will retain the exit code from their last run.
We have to be smart enough not to get the exit code until the container exited.

Reply:
I don't actually know if an exited bool would help here, though... I suppose it would if we reset it every time we started the container.

Reply:
125 does not imply container failed, it is specifically designed to indicate the command/library failed.
docker, buildah, podman, `chroot`  all have this behaviour.

I just think this is a better default then 0, which indicates success.

Reply:
Well podman ps should not report an exit code if the container is running, since it is undefined.  Similarly if the container was never started, just created.  But if the container is ran and did not execute correctly we should not be stating it as 0.  

I will split this out into a separate PR, so we can get this one merged, then we can handle the podman ps problem.